Exiled.API/Features/Door.cs

```diff
@@ -90,7 +90,27 @@ public Door(DoorVariant door, Room room)
         public Room Room { get; }
 
         /// <summary>
-        /// Gets or sets a value indicating whether or not the door is open.
+        /// Gets a value indicating whether or not the door is fully closed.
+        /// </summary>
+        public bool IsFullyClosed => ExactState is 0;
+
+        /// <summary>
+        /// Gets a value indicating whether the door is fully open.
+        /// </summary>
+        public bool IsFullyOpen => ExactState is 1;
+
+        /// <summary>
+        /// Gets a value indicating whether or not the door is open.
+        /// </summary>
+        public bool IsMoving => ExactState is not(0 or 1);
```

Comment:
`The door is open`? Maybe `the door is moving`?

Reply:
Oups fuck